test/addons/argocd/start

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+from drenv import kubectl
+from drenv import commands
+from drenv import cache
+
+
+def deploy(cluster):
```

Comment:
In the function definition, I prefer to use `def deploy_argocd(cluster):` , since this is more generic and could be used to deploy on any cluster. When we do pull mode, we need deploy argocd on all 3 clusters.
The caller will be `deploy_argocd(hub)` so it is clear that currently we only deploy on hub.